src/parser.ts

```diff
@@ -96,12 +96,12 @@ function cleanupYaml(ref: any){
                 ref[key] = {};
                 ref[key][outputKeyName] = outputData;
 
-            }else if(key != 'Attributes' && typeof ref[key] == "object"){
+            }else if(key != 'Attributes' && typeof ref[key] == "object" && ref[key] !== null){
```

Comment:
And here too.

The result of this was that I had to modify the invalid json doc, so that it would also be invalid yaml